fs/rc/rcserver/rcserver.go

```diff
@@ -325,8 +326,12 @@ func (s *Server) serveRemote(w http.ResponseWriter, r *http.Request, path string
 		directory := serve.NewDirectory(path, s.HTMLTemplate)
 		for _, entry := range entries {
 			_, isDir := entry.(fs.Directory)
-			directory.AddEntry(entry.Remote(), isDir)
+			directory.AddHTMLEntry(entry.Remote(), isDir, entry.Size(), entry.ModTime(r.Context()))
```

Comment:
I'm concerned about this line of code..

entry.ModTime() can be expensive to calculate (eg in the S3 and Swift backends it will require a round trip to the server).

With the other changes that was based off the VFS which has a `--no-modtime` so the user can control this.

We don't have that option here although I guess the user could use `--use-server-modtime` which would have the same effect.

What do you think?

Reply:
For my information, is there a reason VFS isn't in rc-server?  Or is it just that it hasn't been added yet?  

We could add the no-modtime to the rc-server which I could then pass the modtime as 'time.Time{}' which is a time of zero effectively disabling it. I've written a function in data.go and added to teh template that will allow me to hide the dates in the template with a —  dash like the other entries if the time is passed like that.  

Additionally if no-modtime is specified even in the http/webdav it would be good I think to pass the time{} to disable it since that is what the user asked for.  I think if no-modtime is passed now in http it seems to use the current start time of the remote.  What do you think about this?

Thoughts?

Reply:
We could also just simply disable modtimes in rc-server since the default will use more transactions.  Easy enough.